app.shazel.pcheetah.json

```diff
@@ -0,0 +1,126 @@
+{
+ "_cli": '
+flatpak-builder --repo=_repo _build _mani
+         ',
+ "app-id":  "app.shazel.pcheetah",
+ "command":            "pcheetah",
+ "sdk":     "org.kde.Sdk",
+ "runtime": "org.kde.Platform",
+ "runtime-version":          "6.6",
+ "finish-args": [
+  "--filesystem=host",
+  "--filesystem=home",
+  "--share=ipc",
+  "--share=network",
+  "--socket=x11",
+  "--socket=wayland",
+  "--socket=pulseaudio",
+  "--device=dri",
+  "--device=all"
+ ],
+ "modules": [
+  {
+   "name": "_meta",
+   "buildsystem": "simple",
+   "build-commands": [
+    "cp -pr share /app"
+   ],
+   "sources": [
+    {"type": "dir", "path": "_meta"}
```

Comment:
Can you also indent the sources block properly everywhere? See the other manifests on Flathub for examples.

Makes everything more readable.

Reply:
now.  i don't like that format of indenting.  it's json - can't i format it as I like it?

is the manifest file considered flathub's or something?  I thought it was mine.
Not sure why I have to follow your style guidelines...
i don't want everything on a million lines and i want related things to line up in a fixed width font.

Reply:
ok ok lemme apologize for my "too much coffee" flamin.
 
I can def make my json more as you expect it.
 
my plan is to kill off this branch tho - and give you a new one called app.pianocheetah.pianocheetah.
 
I'll keep your formatting prefs in the new manifest.
 
I also realized you'll need a bigger app icon - tryin to clean it all up and test with more restrictive permissions so this may take a bit.

Reply:
> my plan is to kill off this branch tho - and give you a new one called app.pianocheetah.pianocheetah.

Just rename the file.